service/matching/physical_task_queue_manager.go

```diff
@@ -53,6 +53,7 @@ import (
 	"go.temporal.io/server/common/testing/testhooks"
 	"go.temporal.io/server/common/worker_versioning"
 	"go.temporal.io/server/service/worker/deployment"
```

Comment:
The fact that this is still here makes me concerned that we're accidentally using the `deployment` package somewhere else? Could you double check that wherever this file uses this package is an appropriate use?